packages/web-api/src/controllers/api-controller.spec.ts

```diff
@@ -0,0 +1,220 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import 'reflect-metadata';
+
+import { Context } from '@azure/functions';
+import { ApiController } from './api-controller';
+
+export class ApiControllerMock extends ApiController {
+    public readonly apiVersion = '1.0';
+    public invoked = false;
+
+    protected invokeImpl(): void {
+        this.invoked = true;
+    }
+}
+
+describe('validateContentType()', () => {
+    it('should validate HTTP POST and PUT only', () => {
+        const context = <Context>(<unknown>{
+            req: {
+                method: 'GET',
+            },
+        });
+        const apiControllerMock = new ApiControllerMock(context);
+        const valid = apiControllerMock.validateContentType();
+        expect(context.res).toEqual(undefined);
+        expect(valid).toEqual(true);
+    });
+
+    it('should validate empty body', () => {
```

Comment:
>validate [](start = 15, length = 8)

instead of naming the function as validate, mention the expected outcome. foreg: should fail for empty body.
or shopuld pass for valid body content etc.

Reply:
updated